bedrock/mozorg/templates/mozorg/about/governance/governance.html

```diff
@@ -18,7 +18,7 @@ <h1 class="mzp-c-article-title">{{_('Governance')}}</h1>
 {% trans roles_url=url('mozorg.about.governance.roles'),
     policies_url=url('mozorg.about.governance.policies.policies'),
     organizations_url=url('mozorg.about.governance.organizations'),
-    discussion_url='//groups.google.com/group/mozilla.governance/topics'%}
+    discussion_url='//groups.google.com/a/mozilla.org/g/governance'%}
```

Comment:
Nit: let's remove the protocol relative URL and make this https explicitly:
```suggestion
    discussion_url='https://groups.google.com/a/mozilla.org/g/governance'%}
```